.github/CODEOWNERS

```diff
@@ -4,4 +4,5 @@
 /x-pack/plugins/apm/  @elastic/apm-ui
 /x-pack/plugins/security/  @elastic/kibana-security
 /x-pack/plugins/spaces/  @elastic/kibana-security
+/x-pack/plugins/canvas/  @elastic/kibana-canvas
```

Comment:
**nit**: Would you be opposed to moving this up between `apm` and `security` to keep this section alphabetical?

Reply:
Sure. According to the docs, order matters, but they're all basically the same here.